snapshots/wash/2024-01-04/nutrients.zip.dvc

```diff
@@ -0,0 +1,31 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Water Information System for Europe (WISE) statistics
+    description: |-
+      The WISE Statistics are aggregated data products for creation of the Freshwater indicators "Oxygen consuming substances in European rivers" and "Nutrients in Freshwater in Europe", which are published at the EEA website.
+    date_published: 2023-07-14
+    title_snapshot: WISE statistics - Nutrients
+
+    # Citation
+    producer: European Environment Agency
+    citation_full: |-
+      European Environment Agency  (2023). WISE statistics - nutrients (1990-2021), 2023.
+    attribution_short: EEA
+    # Files
+    url_main: https://sdi.eea.europa.eu/catalogue/srv/eng/catalog.search#/metadata/271f545b-cdc7-4b18-89fb-1adda5a3f4ef
+    url_download: https://discodata.eea.europa.eu/download/WISE_Indicators/v4r1/AggregatedData_Country
```

Comment:
The link produces a 403: Forbidden error

Reply:
Ah I think the download situation has changed since I first ran the snapshot a month ago, I will write some instructions to find the data.